SYCL/Plugin/enqueue-arg-order-image.cpp

```diff
@@ -277,7 +277,10 @@ int main() {
 
   remind();
   testcopyD2HImage();
-  testcopyH2DImage();
+  // Disabling this part of the test because there is no reliable way to create
+  // two separate context in all cases which is required to trigger PI calls
```

Comment:
Please clarify why there is no way to test H2D while D2H above is OK. How is this connected to "context caching"?